project/Versioning.scala

```diff
@@ -13,10 +13,8 @@ object Versioning {
     * git describe version, so we use git directly
```

Comment:
I think this comment is stale and we could remove it.

Reply:
:+1: